aiosmtpd/tests/test_smtp.py

```diff
@@ -1155,3 +1161,17 @@ def test_close_after_helo(self):
             client.send('HELO example.com\r\n')
             client.sock.shutdown(socket.SHUT_WR)
             self.assertRaises(SMTPServerDisconnected, client.getreply)
+
+
+class TestTimeout(unittest.TestCase):
+    def setUp(self):
+        controller = TimeoutController(Sink)
+        controller.start()
+        self.addCleanup(controller.stop)
+        self.address = (controller.hostname, controller.port)
+
+    def test_timeout(self):
+        with SMTP(*self.address) as client:
+            code, response = client.ehlo('example.com')
+            time.sleep(2)
```

Comment:
2 seconds is too long.
Please use `0.1` in configuration and `0.3` in the test.
They are typical values for asyncio test suite itself, should work fine for aiosmtp as well.